.github/workflows/release.yml

```
@@ -21,9 +21,14 @@ jobs:
         run: |
           git config user.name "$GITHUB_ACTOR"
           git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+     
+     - name: Add helm dependency
+        run: |
+          helm repo add bitnami https://charts.bitnami.com/bitnami
```

According to https://github.com/helm/chart-releaser-action/issues/74#issuecomment-1206111172, a generic version of this would read:

```suggestion
     - name: Add repositories
       run: |
         for dir in $(ls -d charts/*/); do
           helm dependency list $dir 2> /dev/null | tail +2 | head -n -1 | awk '{ print "helm repo add " $1 " " $3 }' | while read cmd; do $cmd; done
         done
```

Would we want to do that instead, in case we wanted to consider other dependencies from other sources than Bitnami at some point?